### ISSUE TITLE
scribefire 4.1 for chrome for mac os x not adding wordpress.com blogs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#740]
```
What's the problem?

i am trying to add my wordpress.com blog —> 
http://mindfulfitnessmovement.wordpress.com

i am using the very latest version of your scribefire plug-in for chrome (i've 
also tested this in safari and had the same issue).

when i input all the correct info into your dialogue box for my wordpress.com 
blog, and ask it to add my blog to the scribefire list, it looks like it 
completes and adds it, and the dialogue goes away, but then the blog is never 
added to the blog drop-down list. i can never write to my wordpress.com blog 
with your scribefire extension for chrome.  this is your latest version of your 
scribefire plug-in extension for chrome for mac os x, version 4.1.  this also 
appears similar to your previously reported issue, #517.


What browser are you using?

google chrome for mac os x, version Version 26.0.1410.43.


What version of ScribeFire are you running?

your latest, version 4.1, from your website. when this did not work the first 
time, i uninstalled it, and downloaded again from your web site to make sure i 
had the latest version. i did, but i still had the exact same issue on 
re-install.
```

-----
Original issue reported on code.google.com by `gent...@dharmatrainer.com` on 27 Mar 2013 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
I've got the same problem. Obviously it's not fixed yet. I'm using archlinux 
with the latest chrome.
```

-----
Original comment by `w...@qiu.es` on 8 Aug 2013 at 4:35

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 4.2 for Chrome
```

-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 4:12

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 4:12
* Changed state: **Fixed**
